### DISCUSSION TITLE
Optional “War Mode” so that there’s an incentive to flag yourself for PvP

### DISCUSSION THREAD

**Discussion by stephenenzor** [chromiecraft/chromiecraft#7653] [Suggestions]
This is not a new idea as it has been implemented on some other servers, but it would be cool if flagging yourself for PvP gave you like a 20% experience boost or some such so that there’s a reason to do it. This would be entirely optional of course and maybe make people disable it in a city or some such.

As long as the reward is kept modest and the feature is kept optional I think this could be nice and increase the pool of folks in the world PvP’ing.

Thoughts? This and an optional “Explorers Contract” for a “slow leveling” challenge mode would be really neat and make me love this great server even more! Thank you for all the work you’ve poured into the project and have a good one.

**Comment by minthes** [comment]
I support any suggestion that boosts xp gained for new players. I have friends from other pservers that would roll on Chromie but they're too put off by the low XP rates compared to other servers.
And I really like the idea of having xp increase done partially through a warmode as a fair amount of players would be interested in that.

**Comment by krutoj87** [comment]
This already was suggested here: https://github.com/orgs/chromiecraft/discussions/7206

And as I already said there, I am absolutely against it, because this wouldn't be any different than "giving players experience boosts for CC points to give players an incentive to Support the server."

Optional gameplay advantages is a line that should never be crossed. Because they aren't Optional. Players not participating in them are degraded to second class players. 
As soon as a war mode will be added to CC I will leave. Because the very reason I joined this server is because I want to play without having to put up with PVP or other players having unfair advantages over me.

**Reply by minthes** [reply]
**Warmode is a trade-off, not an unfair advantage.** It rewards players who choose to forego a safe leveling experience by opting into the risk of open-world PvP. Those who enable War Mode accept the increased danger of being attacked by other players at any time, which balances the extra rewards they receive. Non-participating players are not forced into this risk and can continue progressing in peace without any disadvantage.

The claim that "not participating in War Mode degrades players to second class" misunderstands the system. **Opting out of War Mode is not a penalty—it’s a choice to avoid PvP.** Both paths have clear trade-offs:

- **With War Mode:** Players receive extra rewards but face the constant threat of PvP which could slow down their progress as they have to do corpse runs from world PvP.

- **Without War Mode:** Players enjoy uninterrupted gameplay but forego the additional rewards.

This design respects both playstyles and ensures that neither is inherently superior.
 
Furthermore, the optional nature of War Mode makes it inherently fair. It’s not a paid boost or exclusive feature—it’s accessible to all players equally. PvE players can continue their gameplay exactly as they prefer, untouched by PvP elements. Meanwhile, the inclusion of War Mode broadens the server’s appeal, creating a more challenging and dynamic world without taking anything away from existing players.

**Reply by TotallyNotMyAccount** [reply]
Oh yeah, I remember that discussion quite well, "nice" to see you being back at it again. I stand by everything I wrote in there in general and about your attitude specifically.
Needless to say this idea still has my support.

**Reply by UndoUreche** [reply]
I am the author of the original suggestion and of course I support it. I've expressed my thoughts regarding its implementation in the linked post of I won't go over it here.
With this out of the way, I also remember our wonderful discussion. You argued that any advantage is too valuable to pass up, but also came to the conclusion that having yourself flagged to get it is too terrible, and makes it worthless. And as such, since you personally cannot benefit from it, the option should not exist for others. 
Please do correct me if I misunderstood the last time, and forgive my bluntness but a while has passed since our talk.

**Reply by krutoj87** [reply]
Player A hates open world PVP and player B loves it. This suggestion puts player A into a lose/lose situation. Because now he needs to decide, if he is going to play the game in a way he hates or if he accepts being a second class player, leveling slower than other players.
For player B it's a win/win situation. Because either he gets to do more of what he loves, or if people still don't participate, he will at least level faster than them.

This server is a PVE server it's made for the playstyle of player A. Most of the players on CC will probably be player A. Something that puts the majority of players on the server in a disadvantage for the advantage of a minority shouldn't exist on the server. It shouldn't be hard to understand, why I'm against being put into a lose/lose situation. 

That's really the same reason, why people who don't want to spend money on the game dislike p2w. It's essentially also an optional incentive to support the server and people can choose to not participate in it. But it puts one group of players into an disatvantage and basically a lose/lose situation while puting the others into an advantage.

If you want a war mode make it vanity items like transmogs only, for players who level to max level with PVP turned on. Just like other challenge modes. But there should never be an actual gameplay advantage tied to it!

**Reply by UndoUreche** [reply]
This idea that "player A loves it while player B hates it" needs to be elaborated on. Why does hypothetical player B hate it? The only downside to it is the risk of death. The risk of death, is the risk of a disturbance of progress, an efficiency loss (and hurt feelings). What is an efficiency loss? A downside, a disadvantage.

I'm amazed at the dishonesty of this entire argument: trying to spin war mode as somehow being entirely advantageous to hypothetical player A, while at the same time acknowledging the existence of the disadvantages that make hypothetical player B hate it. And then, in the same breath, disregarding all of it, without acknowledging that player A is taking on the same risk.

And the way you liken it to p2w is especially disgusting. P2w is vile because of the money transaction, because it links real world monetary success to in-game success. Everything else, literally everything else in a game is done through a sacrifice of in-game resources. Do you want your character to gain more power? Do this optional, weekly raid! Is raiding like p2w? No, because it doesn't involve money. Do you want your wotlk character (soon soon) to earn more xp? Equip this heirloom item! Are heirlooms like p2w? No, because they do not involve money. Do you want the better loot? Risk your time in mythics/heroic (soon soon) for better loot! Somehow, that's not p2w either, yet according to you, risking your time in warmode is.

All of this when  warmode was the middle ground chosen by blizzard themselves, to make these two crowds come together.

**Reply by krutoj87** [reply]
> This idea that "player A loves it while player B hates it" needs to be elaborated on. Why does hypothetical player B hate it?

Dude just let it go, it's getting ridiculous now. Some people just hate frickin PVP like some people hate onions, even though in my opinion onions elevate almost every dish that isn't a dessert. There are just different preferences and people who play on PVE servers play there because they most likely don't like PVP. 

You are trying to catch for every straw to make it seem unreasonable if I don't want to engage with PVP on a frickin PVE server and I also don't want others to have advantages over me.
It's so annoying how you try to downplay the downsides this has on PVE players, when it literally doesn't have any downside for you. It puts you in a win/win situation, that's why you advocating so hard for it. 
But the downsides it has on PVE players are real and for they would be enough to just leave the server, despite all the work I've put into all my chars here. Because a server with war mode is a server I don't ever want to play on!

There also doesn't need to be an incentive to play open world PVP on a PVE server, because the whole purpose of a PVE server is to be able to play the game without having to bother about open world PVP at all. Giving advantages to players who play open world PVP defeats this very purpose. And it would make the experience worse for every Player who chose CC specifically because it's a PVE server. 

And since we just go in circles for the second time now just repeating the same things over and over again, I'm out of this conversation. 
I just hope somebody from the dev team would say something on the matter, so I can either have ease of mind, that this will not be implemented or I can stop wasting time on the characters on this server, if it gets implemented.

**Reply by minthes** [reply]
A PvE server doesn’t mean there’s zero PvP—it means PvP is entirely optional. That’s exactly what Warmode offers: an opt-in system for those who want to engage in open-world PvP, while leaving those who prefer PvE completely unaffected. Players who don’t want to PvP can ignore Warmode entirely, and their gameplay experience won’t change. No one is forcing PvP on a PvE server.

**Reply by UndoUreche** [reply]
The funniest thing is that given my class and spec in the current meta, I'd be at a massive disadvantage if this were implemented lol

**Reply by Apathyxia** [reply]
There is no advantage or disadvantage for warmode. YOU are the one thinking you are in a disadvantegous position if someone else would use warmode. It'd be optional. You are trying to justify something to be disadvantageous because you don't like it. 

### You are not competing with anyone while levelling. If you are, then you are playing on a wrong server.

Levelling is super quick already as it is (double xp). Only challenge modes are on 1x rates. 

When it comes to warmode, you probably wouldn't even see a single person in the world before Outland, not necessarily even then due to the low population of the server, thus completely nullifying the threat that warmode would bring.

**Reply by krutoj87** [reply]
Ok I'm going to add one last thing:
**Would you still make the same arguments if we would talk about exp rate increases buyable for Chromiepoints?** 
If not, then you yourself don't even believe in your arguments. 
I mean it would be an optional incentive to support the server and if a faster leveling speed of one player has no effect on the other player, it shouldn't be a problem, right?

>Levelling is super quick already as it is (double xp). Only challenge modes are on 1x rates.

>When it comes to warmode, you probably wouldn't even see a single person in the world before Outland, not necessarily even then due to the low population of the server, thus completely nullifying the threat that warmode would bring.

You do realise, if you really believe in what you write here, it's an argument against a war mode? **Because if what you write is true a war mode won't change anything, so why waste precious dev time on it?**

**Reply by Apathyxia** [reply]
There is a massive difference in BUYING something and having something as an OPT IN. One can be seen as pay-to-win, one as a challenge. Not everyone would have the option to choose the former, but everyone would have the option do the latter.

Also, don't contributors get permanent rested XP? That's already a perk that people normally don't have. 

I am in favour of war mode in general, but I do not see what its purpose would be for the server. I am not against it, but I, in fact, agree with you that it would be a waste of dev time.

**Reply by krutoj87** [reply]
In order for it to be p2w it would need to give one player an advantage over another player. Which is what I'm saying the whole time and what is constantly denied.

Also I don't know all the percs for the different types of contributions, but I can at least say for contributing money you don't get permanent rested XP. Maybe it's for other types of contributions, I don't know. But if it is, it's at least an incentive for something that helps everyone on the server and not just a handful of players who enjoy open world PVP.

A war mode that only rewards vanity items wouldn't be a problem for me. But I don't want gameplay advantages tied to it, because it really only serves a small group of players.

**Reply by Apathyxia** [reply]
You could choose to opt-out is the main thing here. And as I said earlier, if you are considering levelling as a competition, you're on a wrong server. 

What warmode would bring (as suggested), it wouldn't be away from you. I have levelled all my (non-challenge) alts with PvP on and basically never encountered another person, so it's unlikely you'd even be attacked. The whole server revolves around end-game content for now, and levelling provides nothing of interest for majority. 

If anything, IF warmode would bring more people to level up alts or bring new players over, it would just be a net positive for the server. I think that's what matters here. I've heard several people say that their friends would be interested in coming over if we had better xp rates and such - that'd be one way of offering that. 

Even if it's a PvE server, PvP will always be part of the game in one way or another. There is a handful of us that want to see that side of the game be successful as well.
